### PR TITLE
add plugin artifact during publish-utbot workflow

### PR DESCRIPTION
**Pull request by vol0n**
I tested it in my fork in this pr: https://github.com/vol0n/UTBotCpp/pull/4
actions run: https://github.com/vol0n/UTBotCpp/actions/runs/2815892589

### CODE REVIEW

File: docker/action-scripts/build-clion-plugin.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#install java zulu 11 distribution
+sudo apt update -y
+sudo apt install dirmngr --install-recommends -y
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
+sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main' -y
+
+sudo apt update -y
+sudo apt install zulu-11 -y
```
Comment (ladisgin) [inline docker/action-scripts/build-clion-plugin.sh:10]: Why we need zulu instead of open-jdk?

**Review by ladisgin — COMMENTED** [review]
Check that action work correctly

File: docker/action-scripts/build-clion-plugin.sh (hunk 1 above)
Comment (vol0n) [reply docker/action-scripts/build-clion-plugin.sh:10]: Initially github actions for IJ Plugin use zulu (https://github.com/JetBrains/intellij-platform-plugin-template/blob/8188176d4272378939bfcc6b0bfc2c2008c0d2ca/.github/workflows/build.yml#L52), I think open-jdk can be used instead, but did not check it.

Comment (ladisgin) [reply docker/action-scripts/build-clion-plugin.sh:10]: Ok but then use same way to install it. Because add by key don't look like good solution for CI

Comment (vol0n) [reply docker/action-scripts/build-clion-plugin.sh]: fixed

**Review by ladisgin — APPROVED** [review]

**Pull request merged**